cli/printer/text.go

```diff
@@ -338,6 +340,7 @@ func (t *TextPrinter) PrintServiceSummaries(services ...*models.ServiceSummary)
 		rows = append(rows, row)
 	}
 
+	t.StopSpinner()
```

Comment:
perhaps a better way to do this would be to use a single function for `TextPrinter` to actually print to stdout. It basically would just wrap `fmt.Println` but call StopSpinner() first

Reply:
✅